docker/development.yml

```diff
@@ -0,0 +1,7 @@
+version: '3.4'
+
+services:
+  web:
+    build:
+      args:
+        - REACT_APP_API_URL=http://localhost/api
```

Comment:
When docker-compose, these containers are proxied by nginx => no need to specify port number